phpstan-baseline.neon

```diff
@@ -1,1005 +1,51 @@
 parameters:
+	checkMissingIterableValueType: false
```

Comment:
I don't think this parameter should live on the baseline file.. would suggest to move it to `phpstan.neon.dist`